Makefile

```diff
@@ -89,8 +89,9 @@ operator_tag=
 maven_mirror=
 feature=
 local=false
+version=
 run-smoke:
-	./hack/run-smoke.sh -c ${concurrent} --feature ${feature} --ope_name $(operator_image) --ope_tag $(operator_tag) --maven_mirror $(maven_mirror) --local ${local}
+	./hack/run-smoke.sh -c ${concurrent} --feature ${feature} --ope_name $(operator_image) --ope_tag $(operator_tag) --maven_mirror $(maven_mirror) --local ${local} --version ${version}
```

Comment:
Also, I am not a big fan to put too many parameters into the Makefile.
That is why I was keeping some only as env variables, as there are not often used.
Maybe `operator_image`, `operator_tag` and `maven_mirror` could be also removed. So we keep the essential here.
And the rest can be configured via env variables.
@sutaakar @ricardozanini wdyt ?

Reply:
Sounds ok to me.

Reply:
I'd rather having the parameters than environment variables. It's more verbose, since we can exactly see what's happening and that the script is not taking into account any sort of environment var.

Reply:
ok makes sense.
so we would need to extract all env vars as parameters of `run-smoke.sh` and `Makefile`.
Should we create an issue for that ?

Reply:
if you need to keep the env vars to pass information to the smoke test, we can keep them. Otherwise, if we are just having the env vars to pass arguments to the `make` command, then I'd say to open a JIRA for that and add the required parameters to it.

Reply:
anyway we need to pass env vars to the tests.
but if we want the `make` to be fully parametric, we need to update the `Makefile` and `run-smoke.sh` to set the correct env vars for the tests, depending on the given parameters. This is not done for now. Won't take long though.